src/GovUk.Education.ExploreEducationStatistics.Admin/Services/GlossaryService.cs

```diff
@@ -0,0 +1,36 @@
+#nullable enable
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
+{
+    public class GlossaryService : IGlossaryService
+    {
+        private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
+
+        public GlossaryService(IPersistenceHelper<ContentDbContext> persistenceHelper)
+        {
+            _persistenceHelper = persistenceHelper;
+        }
+
+        public async Task<Either<ActionResult, GlossaryEntryViewModel>> GetGlossaryEntry(string slug)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<GlossaryEntry>(query => query
+                    .Where(e => e.Slug == slug))
+                .OnSuccess(e => new GlossaryEntryViewModel
+                {
+                    Title = e.Title,
```

Comment:
Talked to Duncan about this on Slack and we're going to leave this as it is. We do use automapper in other places in the code for situations like this, but we've also had a case or two where bugs have arisen or it's not easy to see how something has been copied and then having to hunt in the automapper config to find out what's going on. I think I'd prefer to pay the cost in terms of boilerplate to avoid those issues.